templates/SilverStripe/Admin/Forms/UsedOnTable.ss

```diff
@@ -0,0 +1 @@
+<div class="used-on__polyfill-holder $extraClass" $AttributesHTML('class')></div>
```

Comment:
You need the `$get` in `$getAttributesHTML()` otherwise it won't work properly. Calling a getter as a method (instead of as a getter) requires the full method name.

Reply:
strange, this works for me without the `$get`
otherwise my entwine polyfill wouldn't have worked...